test/langtools/tools/javac/patterns/SOEDeeplyNestedBlocksTest.java

```diff
@@ -23,1165 +23,39 @@
 
 /**
  * @test
- * @bug 8322992
+ * @bug 8322992 8331030
  * @summary Javac fails with StackOverflowError when compiling deeply nested synchronized blocks
  * @compile SOEDeeplyNestedBlocksTest.java
```

Comment:
This line is not necessary; the file will be compiled if needed by the `@run main`

Reply:
Line removed.